com.unity.render-pipelines.high-definition/Runtime/Material/Lit/ShaderPass/LitSharePass.hlsl

```diff
@@ -14,7 +14,7 @@
 #if defined(_REQUIRE_UV2) || defined(_REQUIRE_UV3) || defined(DYNAMICLIGHTMAP_ON) || defined(DEBUG_DISPLAY) || (SHADERPASS == SHADERPASS_LIGHT_TRANSPORT)
 #define ATTRIBUTES_NEED_TEXCOORD2
 #endif
-#if defined(_REQUIRE_UV3) || defined(DEBUG_DISPLAY)
+#if defined(_REQUIRE_UV3) || defined(DEBUG_DISPLAY) || ((SHADERPASS == SHADERPASS_LIGHT_TRANSPORT) && defined(EDITOR_VISUALIZATION))
```

Comment:
convert all ((SHADERPASS == SHADERPASS_LIGHT_TRANSPORT) && defined(EDITOR_VISUALIZATION)) -> defined(EDITOR_VISUALIZATION))

Reply:
Done in 8611bcaa08